README.md

```diff
@@ -44,13 +44,16 @@ Version 1.0 / 23.May.2022
 
 **Model Architecture:** Modified from Megatron-LM GPT2 ([paper link](https://arxiv.org/abs/1909.08053)):
 
-1.  Layer normalization applied to word embedding layer
-
-2. [ALiBI positional encodings](https://arxiv.org/pdf/2108.12409.pdf)
+* Decoder-only architecture
+* Layer normalization applied to word embeddings layer (`StableEmbedding`; see [code](https://github.com/facebookresearch/bitsandbytes), [paper](https://arxiv.org/pdf/2110.02861.pdf))
+* ALiBI positional encodings ([paper](https://arxiv.org/pdf/2108.12409.pdf)), with GeLU activation functions
+* 176 billion parameters:
+    * 70 layers, 112 attention heads
+    * Hidden layers are 14336-dimensional
```

Comment:
I feel a comma would make the dimensionality easier to read. i.e. `14,336-dimensional`

Reply:
these aren't big numbers, and 2,048 suggested below is definitely looking odd.

Reply:
Committed change.